packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -3,7 +3,6 @@ import ResizeObserver from 'resize-observer-polyfill';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { FormattedMessage } from 'react-intl';
-import Button from 'terra-button';
```

Comment:
Should button be removed from CollapsibleMenuView's package.json? or is it used elsewhere in the package?

Reply:
No, assuming it shouldn’t be removed. (I just pressed the Github Revert button on merged PR.) 
Looking at previous version and will modify, or just may cancel this PR and only change what is needed rather than full revert, since just needs to remove the extra prop that was added.

Reply:
It’s used [here](https://github.com/cerner/terra-framework/blob/master/packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx#L140).

Reply:
Replacing this PR with #966 instead